src/geometry.jl

```diff
@@ -166,24 +191,23 @@ end
 #     # ...
 #     return Geometry(volume, length, (length1, length2, length3), area)
 # end
-function calc_area(shape::Ellipsoid, a, b, c)
-    #e = ((a ^ 2 - c ^ 2) ^ 0.5 ) / a # eccentricity
-    #2 * π * b ^ 2 + 2 * π * (a * b / e) * asin(e)
-    p =  1.6075
-    (4 * π * (((a ^ p * b ^ p + a ^ p * c ^ p + b ^ p * c ^ p)) / 3) ^ (1 / p))
-end
-function calc_area(shape::Ellipsoid, body)
+
+function calc_area(shape::Ellipsoid, body::Body)
     a = body.geometry.lengths[1] / 2
     b = body.geometry.lengths[2] / 2
     c = body.geometry.lengths[3] / 2
+    return calc_area(shape, a, b, c)
+end
+function calc_area(shape::Ellipsoid, a, b, c)
+    #e = ((a ^ 2 - c ^ 2) ^ 0.5 ) / a # eccentricity
+    #2 * π * b ^ 2 + 2 * π * (a * b / e) * asin(e)
     p =  1.6075
-    (4 * π * (((a ^ p * b ^ p + a ^ p * c ^ p + b ^ p * c ^ p)) / 3) ^ (1 / p))
-    # e = ((a ^ 2 - c ^ 2) ^ 0.5 ) / a # eccentricity
-    # 2 * π * b ^ 2 + 2 * π * (a * b / e) * asin(e)
+    return(4 * π * (((a ^ p * b ^ p + a ^ p * c ^ p + b ^ p * c ^ p)) / 3) ^ (1 / p))
 end
 
+# TODO: should we use this rather than the one below that ignores theta?
```

Comment:
for now because that is what is in the NicheMapR ectotherm model, whereas ultimately we'll want to use one with the zenith angle